Hucario version

pvrs12 opened the pull request.

Commit:
another megacommit for untracked changes

Commit:
another megacommit for untracked changes

Commit:
forgot to cache a response

pvrs12 committed:
Merge branch 'master' into hucario-version

Commit:
This should unify, I think

- Removes oldpopup.js again (the newest version only uses newpopup.js)
- Edits anesidora.js to not include getSongBlobUrl function. This was only included in the first place as a hacky backport for functionality that's in this release.

hucario commented:
OK, I think this does it.

hucario commented:
@pvrs12 
Looks like it worked. Sorry it took so long to get done - it didn't take that long once I *got* to it... but getting to it took a while.

Also, it looks like the GitHub action is downloading jQuery - it doesn't need to anymore, so I'll push a change to that once this is merged.

hucario commented:
It's been ready for a while, and I haven't heard any problems from @pvrs12 ... so, if I don't hear anything more for 14 days I'll just push this.

Commit:
Last bugfixes and theming changes

Commit:
Fix a few AMO warnings
- Changed out vendored JSColor for non-minified version so that it wouldn't eval stuff for terseness
- Stripped out eval from the non-minified version (I don't need the loose JSON parsing)
- Added data_collection_permissions to manifest.json to tell AMO that we do not collect user data

Commit:
Versionbump and additional AMO warnings

Commit:
Fix: Waterfox-secific cover positioning bug

Commit:
Fix: Various Android-specific bugs

Commit:
Fix: all web-ext lint errors

Commit:
Add error logging so people can paste it into issues

Commit:
Fix: Re-add HTTPS conversions for art urls (whoops)

Commit:
Add local stats system

Commit:
Version bump

Commit:
Set more sensible defaults

Commit:
*Actually* fix Waterfox cover issue

Commit:
Added back hotkey editing

Commit:
Fix: tab navigation button sizes unresponsive to theme vars

Commit:
Fix: Not skipping after sleep

Commit:
Consistency for viewport scaling on cover

Commit:
Options tidying

Commit:
Reorganize theming settings

Commit:
Version bump

Commit:
Fix: decrement/increment mixup for statistics

Commit:
Update default presets

Commit:
Settings: 3 fixes in one
- Add "reset to defaults" button
- Update styling such that links inherit text color
- Fix dimension inputs not saving to theme

Commit:
Version bump

Commit:
Fix typo in help message

Commit:
Maybe fix some Android errors

Commit:
Save volume in storage

Commit:
Version bump

hucario committed:
Add is_android to chrome's platform_specific.js

hucario committed:
Update README

hucario committed:
Include instructions for reenabling mv2

hucario committed:
Versionbump firefox

hucario committed:
Versionbump Chrome
